src/vertex/angle65/angle65.JSON

```diff
@@ -0,0 +1,194 @@
+{
+  "name": "ANGLE65",
+  "vendorId": "0x9954",
+  "productId": "0x9970",
```

Comment:
Remove V2 definitions as it is not needed (src folder one)

Reply:
Some PCB firmware uses the previous version of QMK, V2 is still necessary. Thanks